src/platform/forms/constants.js

```diff
@@ -209,7 +211,7 @@ export const getAllFormLinks = getAppUrlImpl => {
       '21P-0847-substitute-claimant',
     )}/`,
     [VA_FORM_IDS.FORM_21P_527EZ]: `${getAppUrlImpl('pensions')}/`,
-    [VA_FORM_IDS.FORM_21P_530]: `${getAppUrlImpl('burials')}/`,
+    [VA_FORM_IDS.FORM_21P_530V2]: `${getAppUrlImpl('burials-ez')}`,
```

Comment:
should this have a trailing `/`?

Reply:
Great catch, thank you for that. 
Ready for another pass!